buildSrc/settings.gradle

```diff
@@ -22,6 +22,7 @@ include ":public"
 dependencyResolutionManagement {
     versionCatalogs {
         libs {
+            from(files("../gradle/libs.versions.toml"))
```

Comment:
>  - Problem: In version catalog libs, you can only call the 'from' method a single time.


CI doesn't work, will investigate

Reply:
This is because of the symlinks as well, so this from is applied for each gradle module. It seems, it started to fail because of the Gradle 8.0

Reply:
I've pushed fix, so it will be applied for Windows only